dvc/scm/git.py

```diff
@@ -174,39 +72,39 @@ def ignore(self, path):
             ignore_list = open(gitignore, 'r').readlines()
             filtered = list(filter(lambda x: x.strip() == entry.strip(),
                                    ignore_list))
-            if len(filtered) != 0:
+            if filtered:
                 return
 
         msg = "Adding '{}' to '{}'.".format(os.path.relpath(path),
                                             os.path.relpath(gitignore))
         logger.info(msg)
 
         content = entry
-        if len(ignore_list) > 0:
+        if ignore_list:
             content = '\n' + content
 
-        with open(gitignore, 'a') as fd:
-            fd.write(content)
+        with open(gitignore, 'a') as fobj:
```

Comment:
always used `fd` for "file descriptor", why did pylint complaint about it?

Reply:
I guess it's more correct [fobj](https://docs.python.org/3/glossary.html#term-file-object) because the `open()` built-in actually returns one, so yeah